spec/resources/cdl/automatic_completer_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe CDL::AutomaticCompleter do
+  let(:file) { fixture_file_upload("files/sample.pdf", "application/pdf") }
+  let(:query_service) { Valkyrie.config.metadata_adapter.query_service }
+  let(:change_set_persister) { ScannedResourcesController.change_set_persister }
+  describe ".run" do
+    context "when there's in process CDL items" do
+      it "does not approve them" do
```

Comment:
Can you also change these test names to say "complete" instead of "approve"?

Reply:
Sure, done.